content/react-part-2/reading/conditional-rendering/index.md

```diff
@@ -0,0 +1,452 @@
+---
+title: "Conditional Rendering"
+date: 2023-05-03T11:39:25-05:00
+draft: false
+weight: 2
+originalAuthor: Courtney Frey # to be set by page creator
+originalAuthorGitHub: speudusa # to be set by page creator
+reviewer: Kimberly Horan # to be set by the page reviewer
+reviewerGitHub: codinglikeagirl42 # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+TODO: link to conditionals chapter in JS
```

Comment:
Suggestion -> make this a comment, so you can still find it but it doesn't appear as a line in the book.

Reply:
i think i will pull this for now. if we need a link back, we can add it once the rest of the JS is in the book.